contrib/jenkins_tests/tool.sh

```
@@ -16,60 +16,12 @@ rm -rf $tool_dir
 mkdir -p $tool_dir
 cd $tool_dir
 
-tool_list="daemon"
+# No tools to check
+tool_list=""
```

Shellcheck (0.11.0)

[warning] tool_list appears unused. Verify use (or export if used externally).

(SC2034)